tests/e2e/plugins/scf-test-get-field.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Plugin Name: SCF Test Plugin, Get Field Movie Title
+ * Plugin URI: https://github.com/WordPress/secure-custom-fields
+ * Author: SCF Team
+ *
+ * @package scf-test-plugins
+ */
+
+add_filter( 'the_content', 'scf_add_get_field_at_the_end' );
+
+/**
+ * Add post-formats support to pages
+ */
+function scf_add_get_field_at_the_end() {
+	// Get the field object to validate it exists.
+	$field_object = get_field_object( 'movie_title' );
+
+	// Only proceed if the field exists and is a valid type.
+	if ( $field_object && isset( $field_object['type'] ) && 'text' === $field_object['type'] ) {
+		$field = get_field( 'movie_title' );
+
+		// Ensure we have a string value and sanitize it.
+		$field = is_string( $field ) ? $field : '';
+
+		// Sanitize the field value using WordPress sanitization functions.
+		$field = sanitize_text_field( $field );
+
+		// Escape the output for HTML context.
+		$escaped_field = esc_html( $field );
+
+		// Use wp_kses_post to allow safe HTML if needed, but escape by default.
+		$output = wp_kses_post( '<br><p id="scf-test-movie-title">Movie title: ' . $escaped_field . '</p>' );
```

Comment:
@pkevan Should SCF escape automatically within the get_field function?

Docs says that you need to do this `$escaped_wysiwyg = get_field('wysiwyg', false, true, true);`, but also recommends using `wp_kses_post`.

Am I being redundant here? I have that feeling.

Reply:
> @pkevan Should SCF escape automatically within the get_field function?

Generally this isn't expected, and would more than likely get flagged when using phpcs anyway. 

Escaping should be in the realm of whatever is outputting the data, so in the case of the not needed, but generally any outputting functions within the plugin which requires no user input should be escaping it after using `get_field` i.e. the shortcodes.